lib/run/index.js

```diff
@@ -177,26 +182,29 @@ module.exports = function (options, callback) {
                     });
                 },
 
-                assertion: function (cursor, assertion) {
-                    !assertion && (assertion = {});
+                assertion: function (cursor, assertions) {
 
-                    // store the legacy assertion index
-                    assertion.index && (legacyAssertionIndices[cursor.ref] = assertion.index);
+                    _.forEach(assertions, function (assertion) {
+                        !assertion && (assertion = {});
 
-                    emitter.emit('assertion', (assertion.passed ? null : {
-                        name: _.get(assertion, 'error.name', ''),
-                        index: assertion.index,
-                        message: _.get(assertion, 'error.message', ''),
+                        // store the legacy assertion index
+                        assertion.index && (legacyAssertionIndices[cursor.ref] = assertion.index);
 
-                        stack: 'AssertionFailure: ' + _.get(assertion, 'error.message', '') + '\n' +
-                            '   at Object.eval sandbox-script.js:' + (assertion.index + 1) + ':' +
-                            ((cursor && cursor.position || 0) + 1) + ')'
-                    }), {
-                        cursor: cursor,
-                        assertion: assertion.name,
-                        skipped: assertion.skipped,
-                        error: assertion.error,
-                        item: run.resolveCursor(cursor)
+                        emitter.emit('assertion', (assertion.passed ? null : {
+                            name: _.get(assertion, 'error.name', 'AssertionFailure'),
```

Comment:
@shamasis  Legacy tests do not have `error` on the Assertion. I think `postman-sandbox` should set an `AssertionFailure` error for legacy errors.